### PR TITLE
feat: default masterkey generation

### LINKED ISSUE #2921: Autogenerate Master Key with wallet creation
Part of https://github.com/getAlby/lightning-browser-extension/issues/2917 epic

Master Key should be generated with each wallet creation

### PR DESCRIPTION
**Pull request by pavanjoshi914**
### Describe the changes you have made in this PR

auto masterkey generation during account creation

### Link this PR to an issue [optional]

Fixes  #2921
### Type of change

(Remove other not matching type)

- `feat`: New feature (non-breaking change which adds functionality)

### How has this been tested?

create account. go to account settings see keys generated already

### Checklist

- [X] Self-review of changed code
- [X] Manual testing
- [X] Added automated tests where applicable
- [X] Update Docs & Guides
- For UI-related changes
- [X] Darkmode
- [X] Responsive layout

### CODE REVIEW

File: src/extension/background-script/actions/accounts/add.ts [hunk 1]
```diff
@@ -24,12 +29,34 @@ const add = async (message: MessageAccountAdd) => {
...
+    origin: { internal: true },
+  });
+
   state.setState({ accounts: tmpAccounts });
 
   if (!currentAccountId) {
     state.setState({ currentAccountId: accountId });
   }
 
+  edit({
```
Comment (reneaaron) [inline src/extension/background-script/actions/accounts/add.ts:51]: Why this separate edit call? Just to set the toggle to use the Mnemonic for LNURL-auth?

File: src/extension/background-script/actions/accounts/unlock.ts [hunk 2]
```diff
@@ -31,6 +36,32 @@ const unlock = async (message: MessageAccountUnlock) => {
 
   // if everything is fine we keep the password in memory
   await state.getState().password(password);
+
+  // for existing alby accounts we can't write migiration to auto generate mnemonic because of unavailability of password field.
+  if (!account.mnemonic) {
```
Comment (reneaaron) [inline src/extension/background-script/actions/accounts/unlock.ts:41]: This feels very hacky to me. Why do that automatically?

**Review by reneaaron — CHANGES_REQUESTED** [review]

File: src/extension/background-script/actions/accounts/add.ts (hunk 1 above)
Comment (pavanjoshi914) [reply src/extension/background-script/actions/accounts/add.ts:51]: yup needed to set lnurl-auth for mnemonic

**Review by reneaaron — APPROVED** [review]
tACK

**Pull request merged**